Sources/Fluent/Schema/Builder.swift

```diff
@@ -20,23 +42,47 @@ extension Builder {
     }
 
     public func foreignId<E: Entity>(
+        _ name: String = E.foreignIdKey,
         for entityType: E.Type,
         optional: Bool = false,
-        unique: Bool = false
-    ) {
+        unique: Bool = false,
+        constraint fkCreation: ForeignKeyCreation = .auto
```

Comment:
Passing just the foreign key name here would be sufficient. For those wanting to force a foreign key for a certain field they can easily check the `autoForeignKeys` variable themselves and add a foreign key accordingly.